ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -683,7 +684,8 @@ private static void maybeCreateProcessingLogStream(
     
     try {
       final SimpleKsqlClient internalClient =
-          new ServerInternalKsqlClient(ksqlResource, serviceContext);
+          new ServerInternalKsqlClient(ksqlResource, new KsqlSecurityContext(
+              Optional.empty(), serviceContext));
```

Comment:
Creating the processing log is internal and does not require any authenticated user. Using an Optional.empty() instead.